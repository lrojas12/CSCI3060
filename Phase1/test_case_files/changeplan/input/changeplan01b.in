login
admin
changeplan
Helen
00009
logout
quit