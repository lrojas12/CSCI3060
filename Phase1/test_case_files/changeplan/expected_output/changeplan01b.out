Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
You are currently logged in as an administrator.
Enter account holder’s name:
Enter account number:
The transaction payment plan for account 00009 has been successfully changed to non-student.
You have successfully logged out of the administrator account.